src/ansi-c/compiler_headers/arm_builtin_headers.inc

```diff
@@ -0,0 +1,45 @@
+"// clang-format off\n"
+"void __breakpoint(int val);\n"
+"void __cdp(unsigned int coproc, unsigned int ops, unsigned int regs);\n"
+"void __clrex(void);\n"
+"unsigned char __clz(unsigned int val);\n"
+"unsigned int __current_pc(void);\n"
```

Comment:
Something has gone wrong here, this is an auto-generated file. (There are a few others of this kind added in the latest commit.)